fatx/FATX.py

```diff
@@ -131,33 +135,36 @@ def delete_file(self, de: DirectoryEntry, clean=False):
     def status(self):
         print(self.__str__())
 
+    def _seek(self, dest: int):
+        self.f.seek(self.offset+dest)
```

Comment:
nit: You could alternatively create a virtual file handle which maps to a portion of a real file handle.

Reply:
Thanks for the head-up, I never had subclassed a python standard class before. 💪 It was fun. Not sure if it is really that much better but from an educational pov it absolutely is(which is still the main goal of this project).